src/functionalTest/resources/features/F-204 - Get NoC Questions/common/204_NoC_WithoutEvents_Case_Creation_Token_Creation.td.json

```diff
@@ -0,0 +1,29 @@
+{
+	"_guid_": "204_NoC_WithoutEvents_Case_Creation_With_Org_Policies_Token_Creation",
+  "_extends_" : "Case_Creation_Token_Creation",
+
+  "user": {
+    "username": "befta.master.solicitor.becky@gmail.com",
+    "password": "[[$CCD_BEFTA_MASTER_SOLICITOR_1_PWD]]"
+  },
```

Comment:
If the submit event call that uses the event token is using Richard so should this, see `204_NoC_WithoutEvents_Case_Creation_With_Org_Policies`